src/EnergyPlus/UnitarySystem.cc

```diff
@@ -0,0 +1,12162 @@
+                this->designCoolingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%CoolingCoilType_Num == Coil_CoolingWater .OR. Coil_CoolingWaterDetailed
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate water coil to find operating capacity
+                WaterCoils::SimulateWaterCoilComponents(this->heatingCoilName, FirstHVACIteration, this->heatingCoilIndex, initUnitarySystemsQActual);
+                this->designHeatingCapacity = initUnitarySystemsQActual;
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingWater) THEN
+
+            if (this->heatingCoilType_Num == DataHVACGlobals::Coil_HeatingSteam) {
+
+                //     set water-side mass flow rates
+                Real64 mdot = this->maxHeatCoilFluidFlow;
+                PlantUtilities::SetComponentFlowRate(mdot,
+                                                     this->heatCoilFluidInletNode,
+                                                     this->heatCoilFluidOutletNodeNum,
+                                                     this->heatCoilLoopNum,
+                                                     this->heatCoilLoopSide,
+                                                     this->heatCoilBranchNum,
+                                                     this->heatCoilCompNum);
+                //     simulate steam coil to find operating capacity
+                SteamCoils::SimulateSteamCoilComponents(
+                    this->heatingCoilName,
+                    FirstHVACIteration,
+                    this->heatingCoilIndex,
+                    1.0,
+                    initUnitarySystemsQActual); // QCoilReq, simulate any load > 0 to get max capacity of steam coil
+                this->designHeatingCapacity = SteamCoils::GetCoilCapacity(
+                    DataHVACGlobals::cAllCoilTypes(this->heatingCoilType_Num), this->heatingCoilName, initUnitarySystemsErrorsFound);
+
+            } // from IF(UnitarySystem(UnitarySysNum)%HeatingCoilType_Num == Coil_HeatingSteam) THEN
+            if (this->suppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+
+                //     set steam-side mass flow rates
+  
```

Comment:
There's an awful lot of `this->` going on in here. Do we really need those?

Reply:
I think "this" is something we can discuss on a call. I thought this is what was expected and maybe I got it wrong. From my perspective I don't care either way, I'm just happy it's running and working( well not now since I added in other functions that don't work - e.g., calcSZVAVModel).